stytch/consumer/users/types.go

```diff
@@ -305,18 +305,12 @@ type User struct {
 
 // WebAuthnRegistration:
 type WebAuthnRegistration struct {
-	// WebAuthnRegistrationID: The unique ID for the WebAuthn registration.
 	WebAuthnRegistrationID string `json:"webauthn_registration_id,omitempty"`
 	// Domain: The `domain` on which a WebAuthn registration was started. This will be the domain of your app.
 	Domain string `json:"domain,omitempty"`
 	// UserAgent: The user agent of the User.
-	UserAgent string `json:"user_agent,omitempty"`
-	// Verified: The verified boolean denotes whether or not this send method, e.g. phone number, email
-	// address, etc., has been successfully authenticated by the User.
-	Verified bool `json:"verified,omitempty"`
-	// AuthenticatorType: The `authenticator_type` string displays the requested authenticator type of the
-	// WebAuthn device. The two valid types are "platform" and "cross-platform". If no value is present, the
-	// WebAuthn device was created without an authenticator type preference.
+	UserAgent         string `json:"user_agent,omitempty"`
+	Verified          bool   `json:"verified,omitempty"`
```

Comment:
@ollie-stytch it looks like the `webauthn_registrations` param in the docs doesn't match the proto anymore. Can you take a look?

```
export const webauthn_registrations: ParamWithVariants = {
  description: 'An array that contains a list of all WebAuthn registrations for a given User in the Stytch API.',
  name: 'webauthn_registrations',
  type: 'array[objects]',
  nestedParams: [passkey_registration_id, domain, user_agent, last_authenticated_at],
};
```